alf/experience_replayers/replay_buffer.py

```diff
@@ -270,10 +298,27 @@ def get_batch(self, batch_size, batch_length):
                         non-zero priorities in the buffer.
         """
         with alf.device(self._device):
+            recent_batch_size = 0
+            if self._recent_data_ratio > 0:
+                avg_size = self.total_size / float(
+                    self._num_envs) - batch_length + 1
+                if (avg_size * self._recent_data_ratio >
+                        self._recent_data_steps):
+                    recent_batch_size = math.ceil(
+                        batch_size * self._recent_data_ratio)
+
+            normal_batch_size = batch_size - recent_batch_size
             if self._prioritized_sampling:
-                info = self._prioritized_sample(batch_size, batch_length)
+                info = self._prioritized_sample(normal_batch_size,
+                                                batch_length)
             else:
-                info = self._uniform_sample(batch_size, batch_length)
+                info = self._uniform_sample(normal_batch_size, batch_length)
+
+            if recent_batch_size > 0:
```

Comment:
comment that normal_batch could contain recent data and duplicate with recent_batch, even in the case of w/o replacement sampling?